### PR TITLE
Add Tripetto to the Forms category

### PR DESCRIPTION
**Pull request by oneminch**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -620,7 +620,8 @@ Table of Contents
   * [WaiverStevie.com](https://waiverstevie.com) - Electronic Signature platform with a REST API. Receive notifications with webhooks. Free plan watermarks signed documents, but allows unlimited envelopes + signatures.
   * [Wufoo](https://www.wufoo.com/) - Quick forms to use on websites. Free plan has a limit of 100 submissions each month.
   * [Web3Forms](https://web3forms.com) - Contact forms for Static & JAMStack Websites without writing backend code. Free plan allows Unlimited Forms, Unlimited Domains & 250 Submissions per month.
-  * [Zoho Forms](https://www.zoho.com/forms/) - with 40+ field types, customizable themes, situation-specific templates, and a simple user interface, helps you create beautiful, functional forms for all your needs
+  * [Zoho Forms](https://www.zoho.com/forms/) - with 40+ field types, customizable themes, situation-specific templates, and a simple user interface, helps you create beautiful, functional forms for all your needs.
+  * [Tripetto](https://tripetto.app/) - lets you easily create forms and surveys that are really smart. They can be served as shareable links or even self-hosted.
```
Comment (ripienaar) [inline README.md:624]: It says "Free for now and as long as possible", I am not sure this passes the bar for adding here to be honest, it just seems like another service that will be added today and removed again in 6 months time.

Comment (oneminch) [reply README.md:624]: There's a one time flat fee per-form of $89 if you want to remove their branding. It's not completely free.

Comment (ripienaar) [reply README.md:624]: I am ok with branding, I am not ok with a open ended non commit on having a free tier in future.  Many free services are branded, thats ok.

Comment (oneminch) [reply README.md:624]: Your call then. I just wanted to put it out there.

Comment (ripienaar) [reply README.md:624]: Thank you, but gonna close this one, we see a lot of churn from this kind of unclear statements so going on the cautious side.

**Pull request closed without merging**